core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -175,12 +175,28 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
   .oppia-exploration-summary-tile .avatars-num-minus-one {
     color: white;
     font-family: Roboto;
-    font-size: x-small;
+    font-size: 0.55em;
     font-weight: bold;
-    left: 18px;
+    left: 12px;
     position: absolute;
-    top: 18px;
-    padding-left: 4px;
+    padding-left: 1px;
```

Comment:
I think this ought to be the same as padding-right. I understand this doesn't look so good on your machine, but at this point I think we've done enough work to realize this issue isn't as easy to solve as we want it to be, so I would suggest changing this to 2px and shipping.

My current suspicion is that it has something to do with a 0.5px rounding error. Does it look weird at other zoom levels on your machine (particularly 100%)?